calcom/cal.com
# Fix input on trial accounts

## What does this PR do?



This PR adds the pro input for seats on event types for trail accounts

![image](https://user-images.githubusercontent.com/65426560/180807305-c8b1810d-0c43-4fc8-a9a3-a189be0d08b8.png)


Fixes # (issue)



**Environment**: Staging(main branch) / Production

## Type of change




## How should this be tested?



- Log into a trial account and edit an event type
- Go to the seats option, you should be able to type a number
- Log into a pro account and edit an event type
- Go to the seats option and you should still be able to see the same type of input
